### ISSUE TITLE
Rich attachments editor

### ISSUE THREAD

**Issue by yatesco** [notable/notable#1216]
Hi there,

Is it possible to preview a list of notes that match a search (e.g. all notes with a bunch of tags)?

At the moment the Search only shows the title, but I use tags extensively to build up relationships include lots of pictures, so ideally I would:
 - create a new note
 - embed one or two images
 - link that note to another note/add a tag to that note
 - later on, load that tag and see a preview of all related notes

Being able to quickly scan through a large set of notes with images is the key.

Thanks!

**Comment by fabiospampinato** [comment]
I think this is sort of implemented already, you can navigate back and forth search results and that's the preview. I don't think rendering all notes in their entirety (i.e. with images) in search results would be practical 🤔  Does that not work for your workflow?

**Comment by yatesco** [comment]
Unfortunately not as there maybe hundreds of notes :-). I have many small notes, each note only a paragraph or two, which admittedly is an unusual use case.

**Comment by fabiospampinato** [comment]
I'm not sure what the issue is with the current approach, are you basically saying you'd much rather scroll a long list then press shortcuts for this use case?

**Comment by yatesco** [comment]
I want to see the preview of all the matches in a list I can scroll rather than have to select each result to render the note.

**Comment by fabiospampinato** [comment]
What's the advantage of that over iterating over notes with shortcuts? Just having less shortcuts to trigger?

**Comment by yatesco** [comment]
I'm not being clear :-), but the answer is the same as to why Google will show you previews of all the images that match rather than a list of titles and requires you to select each one, keyboard shortcuts or not.

**Comment by fabiospampinato** [comment]
We could have a dedicated interface for managing attachments globally in the future.

Remaining on the same Google analogy it shows you scrollable search results, but doesn't embed the entire thing below each result, as that would be impractical, right?

**Comment by yatesco** [comment]
yep, but as I probably didn't make clear, each note in my use case equates to a single image with textual notes.

So, imagine I am discussing candlestick trading, I might have a whole bunch of screen captures of "2nd entries", one note for each trade. Searching for "2nd entries" should show me all notes with that tag but inline so I can quickly scan the 10s to 100s of 2nd entry trades I've taken.

**Comment by fabiospampinato** [comment]
I can see the usefulness in that, this use case will be addressed once the rich attachments editor gets implemented. I don't see regular search results displaying images as that would probably be a mess, but search results will get richer too (https://github.com/notable/notable/issues/244).

**Comment by yatesco** [comment]
okeydokes - let me close this now :-). Thanks @fabiospampinato

**Comment by fabiospampinato** [comment]
Let's keep this open so that other people who would also like to see this can upvote it, and I can better keep track of it.
